templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -189,6 +189,15 @@ objects:
     password: ${BROKER_PASS}
```

Comment:
Do we need any changes in ```deploy-local-dev-changes.yaml``` for the local broker?

Reply:
I don't believe so because that template doesn't include the broker config.

Reply:
I don't believe we should go through the trouble of creating an empty secret for the user every time since it won't be used unless someone is going through the trouble of configuring a private registry. In that case they can configure it themself.